src/components/List/List.jsx

```diff
@@ -46,7 +74,12 @@ List.propTypes = {
    * ARIA described by string to reference an id to describe by
    */
   ariaDescribedBy: PropTypes.string,
-  children: PropTypes.any
+  children: PropTypes.any,
```

Comment:
I think children should not be defaultProp as it is not a regular prop

Reply:
We do document children as one of the props in the prop types and default props because it is our library standard.
I did change the children prop type thought to be more accurate .

Reply:
thats the standard we implement in all our comps but I did change the prop type to be more accurate :)